charts/simcore-charts/resource-usage-tracker/values.yaml.gotmpl

```diff
@@ -0,0 +1,150 @@
+# Default values for resource-usage-tracker.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
```

Comment:
could this be replicated?

Reply:
It could be replicated. I don't see any obstacles